app/com/yetu/oauth2provider/views/yetuAuthentication/mails/alreadyRegisteredEmail.scala.html

```diff
@@ -189,8 +189,8 @@ <h6 style="color: #666666; font-family: Source Sans Pro, sans-serif; font-weight
                                                 <tr style="padding: 0; vertical-align: top; text-align: left;">
                                                     <td align="center" style="text-align: center; word-break: break-word; -webkit-hyphens: auto; -moz-hyphens: auto; hyphens: auto; border-collapse: collapse; padding: 0px 0px 10px; vertical-align: top; color: #666666; font-family: Source Sans Pro, sans-serif; font-weight: normal; margin: 0; line-height: 19px; font-size: 14px;">
                                                         <span>&copy; 2015 yetu AG &bull; Version 1.0</span>&nbsp;&nbsp;&nbsp;
-                                                        <a href="http://yetu.me/" target="_blank" style="color: #009900; text-decoration: none;">Privacy Policy</a> -
-                                                        <a href="http://yetu.me/" target="_blank" style="color: #009900; text-decoration: none;">Terms and Conditions</a> -
+                                                        <a href="http://help.yetu.com/privacypolicy" target="_blank" style="color: #009900; text-decoration: none;">Privacy Policy</a> -
+                                                        <a href="http://help.yetu.com/termsandconditions" target="_blank" style="color: #009900; text-decoration: none;">Terms and Conditions</a> -
                                                         <a href="http://yetu.com/imprint/" target="_blank" style="color: #009900; text-decoration: none;">Imprint</a>
```

Comment:
Yeah.. Florian asked me to put this links, and he is aware that is not working :D